### PR TITLE
HPC-8242 - FTS Admin setup

### PR DESCRIPTION
**Pull request by czmj**
FTS Admin rewrite has been started here, in the CDM monorepo, as it relies on the libs in this repository.

The [hpc-apps-core](https://github.com/UN-OCHA/hpc-apps-core) repository was originally created to house these libs and [hpc-ftsadmin-site](https://github.com/UN-OCHA/hpc-ftsadmin-site) repository was created to host FTS Admin rewrite.

[hpc-apps-core](https://github.com/UN-OCHA/hpc-apps-core) was meant to be imported in both CDM (here) and FTS Admin (in [hpc-ftsadmin-site](https://github.com/UN-OCHA/hpc-ftsadmin-site)), but this has not been possible without major changes to CDM and potentially removing Nx from CDM, due to the way Typescript/Nx are set up.

CDM is used in TS config of hpc_service as its [baseUrl is set](https://github.com/UN-OCHA/hpc_service/blob/f37342188c768a2c1de3876dd3db0c85047ded40/tsconfig.json#L16) to external/hpc_cdm, but that prevents other external paths to be included.

Extracting [hpc-apps-core](https://github.com/UN-OCHA/hpc-apps-core) into npm package, like it's done for [hpc-api-core](https://github.com/UN-OCHA/hpc-api-core) may be an option in the future.

### CODE REVIEW

**Review by unocha-hpc — APPROVED** [review]
Checks have passed and this pull request is ready for manual review

**Review by unocha-hpc — APPROVED** [review]
Checks have passed and this pull request is ready for manual review

**Review by unocha-hpc — APPROVED** [review]
Checks have passed and this pull request is ready for manual review

**Review by unocha-hpc — APPROVED** [review]
Checks have passed and this pull request is ready for manual review

**Review by unocha-hpc — APPROVED** [review]
Checks have passed and this pull request is ready for manual review

**Review by unocha-hpc — APPROVED** [review]
Checks have passed and this pull request is ready for manual review

**Review by unocha-hpc — APPROVED** [review]
Checks have passed and this pull request is ready for manual review

File: apps/hpc-ftsadmin/src/i18n/langs/fr.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      "error": "Erreur:",
+      "retry": "Réessayer"
+    },
+    "unsupportedBrowser": {
+      "title": "Navigateur non compatible! ",
+      "info": "Malheureusement, votre présent navigateur n’est pas compatible. Veuillez utiliser Firefox ou Chrome. Pour ce faire, simplement copiez le lien (URL) dessus, ouvrez Firefox, ou Chrome, et collez le lien sur la barre d’adresse du nouveau navigateur."
+    }
+  },
+  "navigation": {
+    "flows": "Search Funding Flows"
```
Comment (Pl217) [inline apps/hpc-ftsadmin/src/i18n/langs/fr.json]: This seems like the only untranslated string.

**Review by Pl217 — CHANGES_REQUESTED** [review]
Let's extract work on updating MUI to a separate PR, because it mostly deals with updating CDM.  Also, that commit (b1ccf86e1deea5a1f77989e4328fa5ac13c22286) reorganizes imports too much, which makes changes harder to read. Please open this as a separate PR, where second commit can make import reordering, if you want.

Second complaint is that 1981dd27efcaef4700fbda732d62726768c5c6f3 makes same changes as #307. Besides me wanting changes requested in that PR, I also don't think committing like this is desirable. Your first commit copies most of the things from CDM to FTS Admin, which includes the tests, which are then "fixed" in 1981dd27efcaef4700fbda732d62726768c5c6f3. I would prefer correct tests from the first commit or do a separate commit which sets up testing in FTS admin app.

Last complaint is that we're starting a new project, adding a new app to the repo, yet no documentation is changed. We need to document that we have utilized this repo for another app, document how to start the server on your local, etc. Also, this PR's first comment should document what we're doing here and how we ended up reusing CDM repo. We opened [hpc-ftsadmin-site](https://github.com/UN-OCHA/hpc-ftsadmin-site) and [hpc-apps-core](https://github.com/UN-OCHA/hpc-apps-core) repos, yet we're still not using them. All of the discussions we had around this should be documented for everyone else (now and in the future) to see.

**Comment by czmj** [comment]
Moving this to Pending prior merge as it depends on HPC-8326 now that has been moved to a separate PR

**Review by unocha-hpc — APPROVED** [review]
Checks have passed and this pull request is ready for manual review

**Review by unocha-hpc — APPROVED** [review]
Checks have passed and this pull request is ready for manual review

**Review by unocha-hpc — APPROVED** [review]
Checks have passed and this pull request is ready for manual review

**Review by unocha-hpc — APPROVED** [review]
Checks have passed and this pull request is ready for manual review

**Review by unocha-hpc — APPROVED** [review]
Checks have passed and this pull request is ready for manual review

**Review by Pl217 — CHANGES_REQUESTED** [review]
Looks to me that rebase was not done perfectly. The first commit is cbf61a656625a8e0efc1802e7d7ba6de54abd116, which is already merged into develop in #315   (8f02d1e7513429b91c5e8192fe5fecc193bfe7f5).

**Review by unocha-hpc — APPROVED** [review]
Checks have passed and this pull request is ready for manual review

**Review by unocha-hpc — APPROVED** [review]
Checks have passed and this pull request is ready for manual review

**Review by Pl217 — APPROVED** [review]

**Pull request merged**